staging/src/k8s.io/api/README.md

```diff
@@ -6,6 +6,29 @@ Schema of the external API types that are served by the Kubernetes API server.
 
 This library is the canonical location of the Kubernetes API definition. Most likely interaction with this repository is as a dependency of client-go.
 
+It is published separately to avoid diamond dependency problems for users who
+depend on more than one of k8s.io/kubernetes/client-go,
```

Comment:
k8s.io/client-go

Reply:
hm, yeah I should probably use the import path. Will wait for other feedback before fixing.